### DISCUSSION TITLE
VS Code tokens to highlight every reactive variable (not only `RefValue`)

### DISCUSSION THREAD

**Discussion by nicooprat** [vuejs/language-tools#2429] [Ideas]
Hi there,

I just discovered that `refs` where underlined in my code this morning:

<img width="702" alt="image" src="https://user-images.githubusercontent.com/645641/220077118-2912b7bb-6e80-4b8a-a254-48772e35ed3d.png">

Would it be possible to make this happen for anything reactive like `ComputedRef` and so on? It would make reading the code way more understandable in terms of reactivity, especially for new comers in this paradigm!

Couldn't find anything in the codebase relative to this, I hope I'm in the right repo :)

Thanks.

**Comment by johnsoncodehk** [comment]
I have just answered it. 👀
https://m.webtoo.ls/@johnsoncodehk/109875438771908609

**Reply by nicooprat** [reply]
I'm not sure to understand. I guess Volar is already working with TypeScript to know that it's a `Ref` (from Vue types). In my example above, I'd like to "highlight" `getSize` and `size`. Would it be worse in terms of performance to apply the same logic to them? Maybe I missed your point though ;)

Edit: conversation from Mastodon for context ⬇️ 

<img width="580" alt="image" src="https://user-images.githubusercontent.com/645641/220129907-bbbf0381-ef66-4cbb-8293-2fb6a40d9cd2.png">

**Comment by johnsoncodehk** [comment]
After re-reading your question, this is a feature of Vetur and not Volar, please forward it to the Vetur repo.
